src/disco/containers/DiscoPane.js

```diff
@@ -72,7 +72,9 @@ export class DiscoPane extends React.Component {
         </header>
         {results.map((item, i) => <Addon {...camelCaseProps(item)} key={i} />)}
         <div className="amo-link">
-          <a href="https://addons.mozilla.org/">{i18n.gettext('See more add-ons!')}</a>
+          <a href="https://addons.mozilla.org/" target="_blank">
```

Comment:
I suppose this is being very paranoid since we own the destination but still. 

*/me puts on tinfoil hat*